index.html

```diff
@@ -65,159 +108,95 @@ <h2>What to Expect at a Hackathon</h2>
             <img src="hackathon_infomatic.png" alt="An infographic hackathon layout">
```

Comment:
Please put img elements in a div so they can be styled as an block level element and it be centered. See line 32 for reference.